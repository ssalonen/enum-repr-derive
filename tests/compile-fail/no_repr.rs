extern crate enum_repr_derive;
use enum_repr_derive::{Into, TryFrom};

#[derive(TryFrom, Into, Copy, Clone, Debug, PartialEq)]
enum Foo {
    VAR1 = -1,
    VAR2 = -3,
}

fn main() {}
