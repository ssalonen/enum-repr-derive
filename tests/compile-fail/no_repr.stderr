error: Repr not found
 --> $DIR/no_repr.rs:4:19
  |
4 | #[derive(TryFrom, Into, Copy, Clone, Debug, PartialEq)]
  |                   ^^^^

error: Repr not found
 --> $DIR/no_repr.rs:4:10
  |
4 | #[derive(TryFrom, Into, Copy, Clone, Debug, PartialEq)]
  |          ^^^^^^^
